node/manifest.test.ts

```diff
@@ -171,6 +167,52 @@ test('Filters out internal in-source configurations in user created functions',
   })
 })
 
+test('excludedPath from ISC goes into function_config, TOML goes into routes', () => {
+  const functions = [{ name: 'customisation', path: '/path/to/customisation.ts' }]
+  const declarations: Declaration[] = [
+    { function: 'customisation', path: '/showcases/*' },
+    { function: 'customisation', path: '/checkout/*', excludedPath: ['/*/terms-and-conditions'] },
+  ]
+  const userFunctionConfig: Record<string, FunctionConfig> = {
+    customisation: {
+      excludedPath: ['/*.css', '/*.jpg'],
+    },
```

Comment:
If the user configures `excludedPath`, does it get saved as an `excluded_pattern`? According to this test, it does: both the declaration and the function config have an `excludedPath` that get converted to `excluded_pattern`. Is it correct to assume that whether the user sets `excludedPath` or `excludedPattern`, it will ultimately be treated the same way?

Reply:
I've understood it the same way.

Reply:
Yes, that's correct. Context in https://github.com/netlify/edge-bundler/pull/402#issuecomment-1560570162.